### PR TITLE
Cf feature addtocalendar

### PR DESCRIPTION
**Pull request by ciaranfontein**
### Cf feature addtocalendar
- Events have comment section
- Comments are mutations
- Comments have user names
- User names come from database based on Id
- Id comes from async storage
- Created user context provider and used consumer in event container
- No link to the event page but the important push is the user context

### CODE REVIEW

File: client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme [hunk 1]
```diff
@@ -55,6 +55,15 @@
...
       shouldUseLaunchSchemeArgsEnv = "YES">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "2D02E47A1E0B4A5D006451C7"
+            BuildableName = "WRFall2019-tvOS.app"
+            BlueprintName = "WRFall2019-tvOS"
+            ReferencedContainer = "container:WRFall2019.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
```
Comment (siddharthparmar7) [inline client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme:66]: did you add them manually?

File: client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
       "size" : "60x60",
+      "idiom" : "iphone",
+      "filename" : "World Logo Copy@5x-60@3x.png",
       "scale" : "3x"
+    },
+    {
+      "size" : "1024x1024",
+      "idiom" : "ios-marketing",
+      "filename" : "World Logo Copy@5x-1024.png",
+      "scale" : "1x"
```
Comment (siddharthparmar7) [inline client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json:55]: Please rename your files to use _ instead of spaces to avoid any future file not found errors.

File: client/js/App.js [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+    comment:
+      'Glad to join the campaign. Love it so much. It inspired me to see the world clearer.',
+    date: 'October 25, 2019',
+  },
+  {
+    user: 'Kelly Clarkson',
+    comment: 'Really like trees',
+    date: 'October 26, 2019',
+  },
+];
```
Comment (siddharthparmar7) [inline client/js/App.js:21]: remove this mock data if not being used.

File: client/js/apollo/index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`,
+          ),
+        );
+      }
+      if (networkError) console.log(`[Network error]: ${networkError}`);
+    }),
+    httpLink,
+  ]),
+  cache: new InMemoryCache(), // Pull data from client-side cache, if available
+});
```
Comment (siddharthparmar7) [inline client/js/apollo/index.js]: Any reason to change your Apollo client now? Your app is not in the production mode and you are not storing cookies on the client so no need for `process.env.NODE_ENV === 'production' ? 'same-origin' : 'include',`.

Please use Apollo boost.

File: client/js/screens/Event/Event.js [hunk 5]
```diff
@@ -1,9 +1,167 @@
...
+const Event = ({
+  title,
+  location,
+  startDate,
+  endDate,
+  description,
+  comments,
+  user,
+}) => {
+  const [value, onChangeText] = React.useState('');
```
Comment (siddharthparmar7) [inline client/js/screens/Event/Event.js]: state should be stored in the container file.

File: client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 6]
```diff
@@ -1,34 +1,58 @@
...
     }
```
Comment (siddharthparmar7) [inline client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json:56]: Make a separate pull request for App logo and Splash screen.

File: client/js/screens/Event/Event.js [hunk 7]
```diff
@@ -1,9 +1,167 @@
...
+    commentItems.push(
+      <View key={i} style={styles.commentContainer}>
+        <Image style={styles.bullet} source={dot} />
+        <Comment
+          user={comments[i].user}
+          comment={comments[i].comment}
+          date={comments[i].date}
+        />
+      </View>,
+    );
```
Comment (siddharthparmar7) [inline client/js/screens/Event/Event.js]: use map method instead of for loop.

File: client/js/screens/Event/Event.js [hunk 8]
```diff
@@ -1,9 +1,167 @@
...
+  }
+
+  //Cleaned up moment formatting calls to keep ternaries readable
+  const startDay = moment(startDate).format('MMM Do, YYYY');
+  const endDay = moment(endDate).format('MMM Do, YYYY');
+  const startTime = moment(startDate).format('h:mma');
+  const endTime = moment(endDate).format('h:mma');
+  console.log('EVENT USER: ', user);
```
Comment (siddharthparmar7) [inline client/js/screens/Event/Event.js]: remove console log.

File: server/config/apollo.js [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+      }
+    },
+    schema
+  });
+
+  apolloServer.applyMiddleware({
+    app,
+    cors: app.get("CORS_CONFIG")
+  });
+};
```
Comment (siddharthparmar7) [inline server/config/apollo.js]: no need to change server. There are no cookies to store. Delete this file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Make a separate branch for all the different features in your project. 

Please make the requested changes. thank you.

File: client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme (hunk 1 above)
Comment (ciaranfontein) [reply client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme:66]: no

Comment (ciaranfontein) [reply client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme:66]: I added app icons through xcode manually

File: client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (ciaranfontein) [reply client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json:55]: These were only used for the app icon and done through xcode with drag and drop based off an online tutorial

File: client/js/App.js (hunk 3 above)
Comment (ciaranfontein) [reply client/js/App.js:21]: Its still being used

File: client/js/screens/Event/Event.js (hunk 5 above)
Comment (ciaranfontein) [reply client/js/screens/Event/Event.js]: Deleted line because form/leaving comments is not part of this merge

File: client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 6 above)
Comment (ciaranfontein) [reply client/ios/WRFall2019/Images.xcassets/AppIcon.appiconset/Contents.json:56]: I did it was merged 3 days ago, not sure why the changes are only showing now

File: client/js/apollo/index.js (hunk 4 above)
Comment (ciaranfontein) [reply client/js/apollo/index.js]: Deleted

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]
Please fix the errors.

**Review by iyuvraajsingh — APPROVED** [review]
Good Work!
